src/components/typography/typography.component.tsx

```diff
@@ -46,214 +53,80 @@ export interface TypographyProps extends SpaceProps, ColorProps {
   display?: string;
   /** Override the list-style-type */
   listStyleType?: string;
-  /** Override the white-space type */
+  /** Override the white-space */
   whiteSpace?: string;
-  /** Override the word-wrap type */
+  /** Override the word-wrap */
   wordWrap?: string;
-  /** Override the text-overflow type */
+  /** Override the text-overflow */
   textOverflow?: string;
   /** Apply truncation */
   truncate?: boolean;
+  /** Override the color style */
+  color?: string;
```

Comment:
Same here, I think `TypographProps extends ColorProps` may be removed

Reply:
Fixed